response_operations_ui/templates/account/activate-account.html

```diff
@@ -63,35 +64,43 @@
                 {% endfor %}
                 {{
                     onsList({
+                        "element": 'ol',
                         "classes": "list--bare",
                         "itemsList": errorData
                     })
                 }}
             {% endcall %}
+            </div>
         {% endif %}
-            <h1>Activate your account</h1>
+            <h1 style="display:inline-block; margin-bottom:1.6rem;">Activate your account</h1>
```

Comment:
I think you can replace this with something like 

```
<h1 class="ons-u-mb-m">Activate your account</h1>
```
The design system has a bunch of utility classes (that aren't documented anywhere that I can find...but they do exist!) but the last character (the m) goes from xs to xl so experiment with what looks right.  Do this change for all the other ones with inline styling that you added as part of this PR

Reply:
I have done the majority of these as recommended. However, the span tag requires inline to put into style, otherwise it ignores the class tag.